connect/connect-element-templates-maven-plugin/element-templates-c7/src/main/java/io/miragon/miranum/connect/elementtemplate/c7/Camunda7ElementTemplateGenerator.java

```diff
@@ -7,16 +7,17 @@
 import io.miragon.miranum.connect.elementtemplate.api.BPMNElementType;
 import io.miragon.miranum.connect.elementtemplate.api.ElementTemplateProperty;
 import io.miragon.miranum.connect.elementtemplate.api.PropertyType;
-import io.miragon.miranum.connect.elementtemplate.impl.ElementTemplateGenerationResult;
-import io.miragon.miranum.connect.elementtemplate.impl.ElementTemplateInfo;
-import io.miragon.miranum.connect.elementtemplate.impl.GenerateElementTemplatePort;
+import io.miragon.miranum.connect.elementtemplate.core.ElementTemplateGenerationResult;
+import io.miragon.miranum.connect.elementtemplate.core.ElementTemplateGenerator;
+import io.miragon.miranum.connect.elementtemplate.core.ElementTemplateInfo;
+import io.miragon.miranum.connect.elementtemplate.core.TargetPlatform;
 import lombok.extern.java.Log;
 
 import java.util.Arrays;
 import java.util.Objects;
 
 @Log
-public class Camunda7ElementTemplateGenerator implements GenerateElementTemplatePort {
+public class Camunda7ElementTemplateGenerator implements ElementTemplateGenerator {
```

Comment:
We should parse the ElementTemplateProperty Annotation in the core module and enrich the ElementTemplateInfo Object with that information.
Moreover the Annotation ElementTemplateProperty should be optional. We should do some kind of conventional parsing of the properties -> Camel case should be separated with spaces and so on.